src/services/tasks.js

```diff
@@ -128,3 +128,30 @@ export const toggleTaskCompletion = async (id, dispatch, setCheckLoading) => {
   }
 };
 
+
+export const updateTags = async (id, tags, dispatch, setTagsLoading) => {
+  const config = getConfig();
+
+  setTagsLoading(true);
+  try {
+    const { data, status } = await axios.put(
+      `${tasksURL}/tags/${id}`,
+      { tags },
+      config
+    );
+
+    if (status === 200) {
+      setTagsLoading(false);
+      console.log("data", data);
```

Comment:
You can remove the console if not necessary.

Reply:
Thanks for pointing out, will remove it in next PR